### DISCUSSION TITLE
S60P flash process?

### DISCUSSION THREAD

**Discussion by mavrc** [Ralim/IronOS#1985] [Q&A]
I got an S60P running the factory 1.07 firmware and I'm interested in installing IronOS, but I'm not sure what's the best way to install it. Am I right in understanding that the only way to install it at present is by flashing through dfu? i.e. https://github.com/Ralim/IronOS-dfu/issues/24#issuecomment-2039101069

**Comment by Ralim** [comment]
Ah; should add it to the docs repo. Its flashed the same way as normal firmware updates for the device. So you connect it to a pc while holding one of the buttons and it will show up in its normal DFU uploader; then you copy and paste the S60P hex file to it and it updates (hopefully).

IronOS-dfu is about replacing the vendor updater with one that we control & can fix bugs in as most of the vendor "copy file to flash disk" built in updaters have bugs and compatibility issues.
